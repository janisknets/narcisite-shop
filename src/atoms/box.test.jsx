import React from 'react';
import renderer from 'react-test-renderer';
import Box from './box';

it('Box renders correctly', () => {
  const tree = renderer
    .create(<Box />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
