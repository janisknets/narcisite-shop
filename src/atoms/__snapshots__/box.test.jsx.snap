// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders correctly 1`] = `
<button
  onClick={[Function]}
  style={null}
  type="button"
>
  Button
</button>
`;
